codespell_lib/data/dictionary_en-GB_to_en-US.txt

```diff
@@ -78,6 +79,8 @@ emphasise->emphasize
 emphasised->emphasized
 emphasises->emphasizes
 emphasising->emphasizing
+enamour->enamor
+enamoured->enamored
```

Comment:
Maybe add `enamours->enamors`, `enamouring->enamoring`?

Reply:
On the other hand, they seem to be missing from GitHub, except from lists of British words or a British poem.